Source/WebCore/svg/graphics/filters/SVGFilter.cpp

```diff
@@ -233,15 +232,14 @@ OptionSet<FilterRenderingMode> SVGFilter::supportedFilterRenderingModes() const
 
 FilterEffectVector SVGFilter::effectsOfType(FilterFunction::Type filterType) const
 {
-    HashSet<Ref<FilterEffect>> effects;
+    FilterEffectVector effects;
 
-    for (auto& term : m_expression) {
-        auto& effect = m_effects[term.index];
+    for (auto& effect : m_effects) {
         if (effect->filterType() == filterType)
-            effects.add(effect);
+            effects.append(effect);
     }
 
-    return copyToVector(effects);
+    return effects;
```

Comment:
Seems unrelated to the purpose of this bug, so should probably be in a separate PR.

Reply:
Okay. Removed.

Reply:
https://github.com/WebKit/WebKit/pull/14596